test/data/TestArgsParse.default.json

```diff
@@ -1 +1 @@
-{"NRTupdater":{"NoPublish":false,"Oneshot":false,"Hostname":"TEST_NODE","Namespace":""},"Resourcemonitor":{"Namespace":"","SysfsRoot":"/sys","ExcludeList":{"ExcludeList":null},"RefreshNodeResources":false},"RTE":{"Debug":false,"ReferenceContainer":{"Namespace":"TEST_NS","PodName":"TEST_POD","ContainerName":"TEST_CONT"},"TopologyManagerPolicy":"","TopologyManagerScope":"container","KubeletConfigFile":"/podresources/config.yaml","KubeletStateDirs":[""],"PodResourcesSocketPath":"unix:///podresources/kubelet.sock","SleepInterval":60000000000},"Version":false,"LocalArgs":{"SysConf":{"ReservedCPUs":"","ResourceMapping":null}}}
\ No newline at end of file
+{"NRTupdater":{"NoPublish":false,"Oneshot":false,"Hostname":"TEST_NODE"},"Resourcemonitor":{"Namespace":"","SysfsRoot":"/sys","ExcludeList":{"ExcludeList":null},"RefreshNodeResources":false},"RTE":{"Debug":false,"ReferenceContainer":{"Namespace":"TEST_NS","PodName":"TEST_POD","ContainerName":"TEST_CONT"},"TopologyManagerPolicy":"","TopologyManagerScope":"container","KubeletConfigFile":"/podresources/config.yaml","KubeletStateDirs":[""],"PodResourcesSocketPath":"unix:///podresources/kubelet.sock","SleepInterval":60000000000,"PodReadinessEnable":true,"NotifyFilePath":""},"Version":false,"LocalArgs":{"SysConf":{"ReservedCPUs":"","ResourceMapping":null}}}
```

Comment:
new line at EOF needed here?

Reply:
nope, it's automatically generate, so it's ok